normandy/selfrepair/static/js/self_repair_driver.js

```diff
@@ -46,10 +46,42 @@ let registeredActions = {};
 window.Normandy = {
     registerAction(name, func) {
         registeredActions[name] = func;
-    }
+    },
+
+    heartbeatCallbacks: [],
```

Comment:
The curmudgeon in me points out that putting this hear makes it available to actions, which probably isn't what we want to do.